src/objects/settings.ts

```diff
@@ -2,12 +2,23 @@ import { addSeconds } from 'date-fns'
 import { reactive } from 'vue'
 import { DEFAULT_LINEAR_COLUMN_SIZE_PIXELS, DEFAULT_TIME_COLUMN_SIZE_PIXELS, DEFAULT_TIME_COLUMN_SPAN_SECONDS } from '@/consts'
 import { HorizontalMode, LayoutSettings, VerticalMode } from '@/models/layout'
-import { emitter } from '@/objects/events'
+import { EventKey, emitter, waitForEvent } from '@/objects/events'
+import { waitForRunData } from '@/objects/nodes'
+
+export async function startSettings(): Promise<void> {
+  const data = await waitForRunData()
+
+  setHorizontalScale(1)
```

Comment:
Now we have a place where we can set the initial horizontal scale. Next PR will set this based on the length of the run data